targets/pna_nic/main.cpp

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2024 Marvell Technology, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+/*
+ * Rupesh Chiluka (rchiluka@marvell.com)
```

Comment:
Do you still have access to this email? If not, maybe your personal email can be used here instead?

Reply:
I can still use GitHub account associated with it.